packages/codegen-ui/lib/generate-form-definition/helpers/form-field.ts

```diff
@@ -52,6 +52,7 @@ export function getFormDefinitionInputElement(
       formDefinitionElement = {
         componentType: 'TextField',
         props: {
+          name: config.inputType?.name || baseConfig?.inputType?.name || FORM_DEFINITION_DEFAULTS.field.inputType.name,
```

Comment:
Added name prop for the field component to have reference to its field in data model during form codegen

Reply:
You can find the name as the key for the element. In the definition. You can pass that in if that's not getting passed in now. "name" is actually a prop on some elements with a separate meaning. I think we should reserve "props" for things that are actually going to be mapped as props on the ui-react component.

Reply:
Thanks I was worried about that. Ill have to find a way to connect the form definition to the mapped studio component